src/playasophy/wonderdome/mode/strip_eq.clj

```diff
@@ -0,0 +1,75 @@
+(ns playasophy.wonderdome.mode.strip_eq
+  (:require
+    [playasophy.wonderdome.geometry.sphere :as sphere]
+    [playasophy.wonderdome.mode.core :as mode]
+    [playasophy.wonderdome.util.color :as color]
+  )
+)
+
+; StripEQ will process audio signals and turn each strip into an indicator for
+; the strength of a range of frequencies over time.
```

Comment:
For a file-level comment like this, use two semicolons (`;;`) so that [marginalia](https://fogus.github.io/marginalia/) will pick them up.